src/components/pages/Contacts/index.tsx

```diff
@@ -117,6 +116,20 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
     page_id = 'faded';
   }
 
+  const handleScroll = (e: UIEvent) => {
+    const tableDiv = document.getElementById('tableDiv');
+    // const org = user.user.org;
+
+    if (tableDiv) {
+      const isBottom =
+        tableDiv.scrollHeight - tableDiv.scrollTop === tableDiv.clientHeight;
```

Comment:
woww, this is awesome!!

Reply:
do we display any feedback to the user while the bottom is loading?

Reply:
maybe we can show spinner while it loads if we're not!

Reply:
Right now I have it showing the wheel as it did when loading all the contacts before

Reply:
But this does cover all the contacts while the new contacts are loading (which in prod could be a non-trivial amounts of time) so maybe we want to change this so the wheel displays below the contacts that have already been loaded

Reply:
oh so it just renders on top of the table?
ye! I think generalizing it to render below contacts makes sense. But we can do it as a fast follow if needed

I am also curious how we should handle contact searches.
Like if someone looks up a contact that's not being rendered it just doesn't show up. 
We might need to create an endpoint in the future to search for specific contacts and concat them to the array of contacts

Will open issue for this!

Reply:
yeah that makes a lot of sense - We can do a fast follow PR for loading wheel at bottom - maybe baked into a PR to generalize the logic for the other 2 identified uses?